src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -2178,8 +2207,14 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
             let data    = Rc::new(sample_data_generator.generate_data()); // FIXME: why rc?
             let content = serde_json::to_value(data).unwrap();
             let data    = visualization::Data::from(content);
-            inputs.set_visualization_data.emit((*node_id,data));
+            inputs.set_visualization_data.emit((*node_id,data.clone()));
```

Comment:
Why we suddenly need to clone data here?

Reply:
Of course we dont :) Just left it and didn't catch it